content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+=========================
+Engineering change orders
+=========================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |BOMs| replace:: :abbr:`BoMs (Bills of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+.. _plm/eco:
+
+Utilize *Engineering Change Orders* (*ECOs*) to track and implement changes to products, and
+:ref:`bills of materials <manufacturing/management/bill-configuration>`.
+
+View and create :abbr:`ECOs (engineering change orders)` within each *ECO type*, a project pipeline
```

Comment:
the short-hand notations at the top of the doc have the acronym definitions in title case, but here's it's not. Let's be consistent 😉

Also why not use the notation here?
```suggestion
View and create |ECOs| within each *ECO type*, a project pipeline
```